src/index.html

```diff
@@ -11,7 +11,190 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="field">
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input 
+            type="text" 
+            name="surname"
+            id="surname"
+            class="input"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">    
+          Name:
+          <input 
+            type="text" 
+            name="name"
+            class="input"
+            id="name"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input 
+            type="number" 
+            name="age"
+            class="input"
+            id="age"
+            value="12" 
+            min="1" 
+            max="100"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input 
+            type="date" 
+            name="birth"
+            class="input"
+            id="dateOfBirth"
+          >
+        </label>
+
+        <label>
+          I accept the term of the agreement
+          <input 
+            type="checkbox" 
+            name="agreement" 
+            id="agreement"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="field">
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input 
+            type="email" 
+            name="email"
+            id="email"
+            class="input"
+            placeholder="email@example.com" 
+            required
+          >
+        </label>
+        <label>
```

Comment:
Add an empty line everywhere where needed
```suggestion
        </label>
        
        <label>
```